Added *lots* of german words

riker09 opened the pull request:
I tried to watch out for profanity but I cannot guarantee that some fowl language has slipped through.

riker09 committed:
Added *lots* of german words

I tried to watch out for profanity but I cannot guarantee that some fowl language has slipped through.

killercup commented:
Thanks for the contribution! I don't really wanna replace the list of words I picked personally with a random list based on usage. If you think this is a category you'd use, can you add it as a new one? Maybe called "Common German words"?

riker09 committed:
Revert changes to original German words category

riker09 committed:
Add common german words as their own category

riker09 commented:
Thanks for taking a look at my changes. I reverted your German Words category back and added my own. I definitly think that this is some I use since my team agreed on the naming schema "English Adjective" and "German Word" (preferrably a noun) together with "alliteration: on" for naming our releases. The motivation behind this was that with only a small list there is a lot of repitition.

killercup commented on categories.js:
```diff
@@ -478,7 +478,7 @@ var categories = {
       "bitartrate",
       "thiophosphate",
-      "crystal ",
+      "crystal ",
```
```suggestion
      "crystal",
```

Can you revert this space? :)

riker09 replied:
My editor must have done this by itself, I didn't touch that line and it must have slipped.

killercup reviewed (commented):
Thanks for the quick update! One small thing and then it's good to go

riker09 committed:
Revert removed space char

killercup commented:
Thanks!
